Lib/test/test_tools/test_makefile.py

```diff
@@ -70,6 +70,7 @@ def test_makefile_test_folders(self):
         # Don't check the wheel dir when Python is built --with-wheel-pkg-dir
         if sysconfig.get_config_var('WHEEL_PKG_DIR'):
             test_dirs.remove('test/wheeldata')
+            used.remove('test/wheeldata')
```

Comment:
I have yet to test this, but won't this KeyError when the thing is not there? Should it use `discard` instead?

Reply:
Ah, yes, absolutely.